packages/@vue/cli-plugin-eslint/index.js

```diff
@@ -41,7 +41,7 @@ module.exports = (api, options) => {
             .add(/node_modules/)
             .add(path.dirname(require.resolve('@vue/cli-service')))
             .end()
-          .test(/\.(vue|(j|t)sx?)$/)
+          .test(/\.(vue|(j|t)sx?|coffee)$/)
```

Comment:
This should also be the responsibility of the `coffee` plugin.
Can be achieved with this statement in `chainWebpack`:

```js
config.module.rule('eslint').set('test', /\.(vue|(j|t)sx?|coffee)$/)
```